docs/master/federation/entity-representations.md

```diff
@@ -0,0 +1,85 @@
+# Entity representation
+
+In order for a subgraph to be able to return an entity from another subgraph, it is necessary to define in the first
+subgraph the representation of the required entity from those fields that are known to the first subgraph.
+Read more about the entity representation in the [Apollo Federation docs](https://www.apollographql.com/docs/federation/v1/entities/#entity-representations).
+
+A representation always consists of:
+- A `__typename` field;
+- Values for the entity's primary key fields.
+
+In this way, the resolver should return an array consisting of the `__typename` field, which corresponds to the entity's
+name, and the primary key that can identify the entity.
+
+### Example 1
+
+In this example, the `review-service` has an entity called `Review`, which in turn has an entity called `Product`
+defined in a separate `product-service`. The relationship between the review and the product is one-to-one.
```

Comment:
```suggestion
defined in a separate product service. The relationship between `Review` and `Product` one-to-one.
```

Aside from formatting, one-to-one does not really make sense here. I would assume a product can have multiple reviews.